signing_service/src/signing_service.py

```diff
@@ -126,6 +127,40 @@ def _validate_arguments(self):
 
 
 def _parse_arguments() -> argparse.Namespace:
+
+    def get_secret_variable_wrapper(env_variable_name=None, file=False, base64_convert=False):
+        def wrapper(**kwargs):
+            return GetSecretVariable(env_variable_name, file, base64_convert, **kwargs)
```

Comment:
Why do you need this wrapper? It does not do anything special. Why not just call the constructor directly?

Reply:
Because this wrapper passing all arguments which I don't have to pass again in constructor like `option_strings`, `dest` or `nargs`. If I removed this wrapper then place where should I call this class would be very big piece of code - using wrapper I can do same thing in one line.